### PR TITLE
feat(WEBRTC-176): Add a quickstart on README

### PR DESCRIPTION
**Pull request by DeividVeloso**
Create steps with how to set up a call center app

## ✋ Manual testing

1. Review the content 
2. See the preview mode of the README.

### Desktop
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

### Mobile
- [ ] Chrome (Android)
- [ ] Safari (iOS)

## 📸 Screenshots
image

### CODE REVIEW

File: call-center/README.md [hunk 1]
```diff
@@ -1,11 +1,103 @@
...
+Head to [telnyx.com/sign-up](https://telnyx.com/sign-up) to sign up for your free Telnyx account.\
+It’ll give you access to our Mission Control Portal where you can buy phone numbers, set up and manage Call Control Applications, and more.
+
+---
+## Step 2: Buy a phone number  
+
+Access Telnyx Mission Control and [buy a phone number](https://portal.telnyx.com/#/app/numbers/search-numbers).\
+This is your Call Center phone number that end users will call to reach your application.
+
+![buy a phone number](https://raw.githubusercontent.com/team-telnyx/telnyx-client-examples/WEBRTC-176/call-center/screenshots/buy_phone_number.png)
```
Comment (DeividVeloso) [inline call-center/README.md:31]: Before the merge, we need to change all places that are using in the URL images from `WEBRTC-176/` to `main`

Comment (SuaYoo) [reply call-center/README.md:31]: You can use relative paths here: `screenshots/buy_phone_number.png`

File: call-center/README.md [hunk 2]
```diff
@@ -1,11 +1,103 @@
-# Call Center
+# Call Center App
+
+This is an example of a call center app. You can use to manage, make and receive calls, create call conferences and control the agents.
```
Comment (SuaYoo) [inline call-center/README.md:3]: Suggested wording tweak:
```suggestion
This is an example of a call center app that uses the Telnyx Client and Call Control SDKs. You can make and receive phone calls in the browser, create call conferences, and manage call center agents.
```

File: call-center/README.md [hunk 3]
```diff
@@ -1,11 +1,103 @@
...
 
 ## Quick start
 
-```sh
-npm install
-npm run setup
-npm start
-```
+In this guide, you’ll learn how to get started with Call Center App using Call Control and Telnyx Portal. Just follow these steps:
```
Comment (SuaYoo) [inline call-center/README.md:7]:
```suggestion
In this guide, you’ll learn how to get started with the Call Center App by using Telnyx SDKs and the Telnyx Portal. Just follow these steps:
```

File: call-center/README.md [hunk 4]
```diff
@@ -1,11 +1,103 @@
...
+
+1. Sign up for a Telnyx Account.
```
Comment (SuaYoo) [inline call-center/README.md:9]:
```suggestion
1. Sign up for a Telnyx Account
```

File: call-center/README.md [hunk 5]
```diff
@@ -1,11 +1,103 @@
...
+2. Buy a phone number
+3. Install Ngrok
+4. Create a Call Control App
+5. Update your phone number
+6. Create a SIP connection
```
Comment (SuaYoo) [inline call-center/README.md:14]:
```suggestion
6. Create a SIP Connection
```

File: call-center/README.md [hunk 6]
```diff
@@ -1,11 +1,103 @@
...
+Access Telnyx Mission Control and [buy a phone number](https://portal.telnyx.com/#/app/numbers/search-numbers).\
+This is your Call Center phone number that end users will call to reach your application.
+
+![buy a phone number](https://raw.githubusercontent.com/team-telnyx/telnyx-client-examples/WEBRTC-176/call-center/screenshots/buy_phone_number.png)
+---
+## Step 3: Install Ngrok
+
+Download and install [ngrok](https://ngrok.com/).\
+Start up ngrok with `ngrok http 8000` and make note of the https `Forwarding` URL.\
+You will need it to be able to receive websocket request from Call Control App
```
Comment (SuaYoo) [inline call-center/README.md:37]:
```suggestion
You will need it to be able to receive websocket requests from the Call Control App 
```

File: call-center/README.md [hunk 7]
```diff
@@ -1,11 +1,103 @@
...
+## Step 8: Write some code
+
+1. (call-center) Run the command `npm run setup` to generate your `.env` file.
+2. (server) Update your `.env` file with information noted from previous steps.
+
+---
+## Step 9: Run the code
+
+1. Navigate into the `call-center` folder
+1. Run `npm install`
```
Comment (SuaYoo) [inline call-center/README.md:82]:
```suggestion
1. Run `npm install` from the `call-center` folder
```

File: call-center/README.md [hunk 8]
```diff
@@ -1,11 +1,103 @@
...
+2. Run `npm start`\
+This will install and start both the server and web apps.\
+This will start an Express app on port specified as the `TELNYX_SERVER_APP_PORT` environment variable (defaults to `8000`).
+3. Open the ngrok Web Interface (ex: <http://127.0.0.1:4040/inspect/http>). You should see `/login` request.
```
Comment (SuaYoo) [inline call-center/README.md:86]:
```suggestion
3. Log in as a call center agent in the browser. Open the ngrok Web Interface (ex: <http://127.0.0.1:4040/inspect/http>) in a separate tab. You should see `/login` request.
```

File: call-center/README.md [hunk 9]
```diff
@@ -1,11 +1,103 @@
...
+4. Call your Call Center phone number from your personal device or a web dialer. You should see requests to `POST /calls/callbacks/call-control-app` come through. (Note: Events may not show up in the expected order, e.g. `initiated` may be logged after `call`.)
```
Comment (SuaYoo) [inline call-center/README.md:87]:
```suggestion
4. Call your Call Center phone number from your personal device or a web dialer. You should see requests to `POST /calls/callbacks/call-control-app` come through.
```

**Review by SuaYoo — CHANGES_REQUESTED** [review]
General suggestions:
- It looks like there's an extra backslash at the end of some lines, ex: `sign up for your free Telnyx account.\`
- Switch to relative file paths for all screenshots

**Comment by DeividVeloso** [comment]
> General suggestions:
> 
> * It looks like there's an extra backslash at the end of some lines, ex: `sign up for your free Telnyx account.\`
> * Switch to relative file paths for all screenshots

I put the backslash to do a new line.

**Comment by SuaYoo** [comment]
> I put the backslash to do a new line.

It doesn't look like this is supported in all markdown processors (in fact, the VSCode default preview still shows backslashes.) Can you replace with two spaces? See: https://www.markdownguide.org/basic-syntax/#line-break-best-practices

**Review by SuaYoo — APPROVED** [review]

**Pull request merged**